src/node_counters.cc

```diff
@@ -65,7 +65,8 @@ static void counter_gc_start(Isolate* isolate,
 static void counter_gc_done(Isolate* isolate,
                             GCType type,
                             GCCallbackFlags flags) {
-  uint64_t endgc = NODE_COUNT_GET_GC_RAWTIME();
+  uint64_t endgc = 0;
```

Comment:
it is just for static C/C++ code analysis application giving errors like it is an uninitialized variable.